sw/device/silicon_creator/rom/data/rom_e2e_testplan.hjson

```diff
@@ -843,7 +843,11 @@
             '''
       tags: ["rom", "verilator", "dv", "fpga", "silicon"]
       stage: V2
-      tests: []
+      tests: [
+        "//sw/device/examples/sram_program:sram_program_fpga_cw310_test_otp_dev",
+        "//sw/device/examples/sram_program:sram_program_fpga_cw310_test_otp_rma",
+        "//sw/device/examples/sram_program:sram_program_fpga_cw310_test_otp_test_unlocked0"
+      ]
```

Comment:
nit: Would you mind adding a `test_suite` that groups all these tests (like [this](https://cs.opensource.google/opentitan/opentitan/+/master:sw/device/silicon_creator/rom/e2e/BUILD;drc=4a56490a55d8d6e53a831b4f169b0078c22b586e;l=376) for example)? This way we won't need to touch this file if we need to add/remove individual testpoints.

Reply:
Good idea. Done!